code/noRAM/app/src/main/java/com/example/noram/OrganizerEventAttendeeListActivity.java

```diff
@@ -179,10 +179,6 @@ public void afterTextChanged(Editable editable) {
                 // Flip showing
                 showing = Showing.SIGNEDUP;
 
-                // Update buttons
-                signedUpButton.setBackgroundColor(getResources().getColor(R.color.light_grey, getTheme()));
```

Comment:
not a problem if you aren't, but if you still want to have the background colors change you can